dist/bin/common

```diff
@@ -155,8 +155,64 @@ SBT_INTF=$(find_lib "*compiler-interface*")
 JLINE_READER=$(find_lib "*jline-reader-3*")
 JLINE_TERMINAL=$(find_lib "*jline-terminal-3*")
 JLINE_TERMINAL_JNA=$(find_lib "*jline-terminal-jna-3*")
-[[ $conemu ]] || JNA=$(find_lib "*jna-5*")
+[[ ${conemu-} ]] || JNA=$(find_lib "*jna-5*")
 
 # debug
 
 DEBUG_STR=-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005
+
+classpathArgs () {
```

Comment:
Maybe rename it to reflect that it's used for the compiler JVM?

Reply:
@liufengyun - good idea.    I didn't see this until after I pushed changes, so we'll have to do that when we update the two other calls to `bin/scalac`.